packages/fxa-content-server/app/styles/modules/_settings.scss

```diff
@@ -396,14 +396,6 @@ section.modal-panel {
   .client-name {
     line-height: 1.2;
 
-    html[dir='ltr'] & {
```

Comment:
This is needed because really really long device names end up being covered by the disconnect button.

<img width="575" alt="Screenshot 2019-07-22 at 15 35 08" src="https://user-images.githubusercontent.com/848085/61641064-57d29080-ac96-11e9-8b22-cf8cff3bc86e.png">

Reply:
Changes made as requested.